### ISSUE TITLE
When training fast RCNN with pretrained, the loss diverges

### ISSUE THREAD

**Issue by Zhang-ren** [zhanghang1989/ResNeSt#49]
When training fast RCNN with pretrained, the first loss value is huge ,and then it bigger and bigger ,what should I do?

**Comment by chongruo** [comment]
Can you provide more details? Which framework are you using? What is config.yaml?

**Comment by zhanghang1989** [comment]
We have released our implementation based on detectron at https://github.com/zhanghang1989/detectron2-ResNeSt
